src/styles/sidebar/components/_index.scss

```diff
@@ -2,7 +2,7 @@
 
 // UI (Preact) Components
 // ----------
-@use './StyledText';
+@use '../../../annotations-ui/StyledText';
```

Comment:
This will become something like `@hypothesis/annotations-ui/StyñedText.css` by defining an export in the library.